crates/re_viewer/src/app.rs

```diff
@@ -1348,6 +1348,39 @@ fn frame_time_label_ui(ui: &mut egui::Ui, app: &mut App) {
 }
 
 fn memory_use_label_ui(ui: &mut egui::Ui, gpu_resource_stats: &WgpuResourcePoolStatistics) {
+    const CODE: &str = "use re_memory::AccountingAllocator;\n\
+                        #[global_allocator]\n\
+                        static GLOBAL: AccountingAllocator<std::alloc::System> =\n    \
+                            AccountingAllocator::new(std::alloc::System);";
+
+    fn click_to_copy(
+        ui: &mut egui::Ui,
+        text: impl Into<String>,
+        add_contents_on_hover: impl FnOnce(&mut egui::Ui),
+    ) {
+        #[allow(clippy::blocks_in_if_conditions)]
+        let text = text.into();
+        if ui
+            .add(
+                egui::Label::new(
+                    egui::RichText::new(text)
+                        .monospace()
+                        .color(ui.visuals().weak_text_color()),
+                )
+                .sense(egui::Sense::click()),
+            )
+            .on_hover_ui(|ui| add_contents_on_hover(ui))
+            .clicked()
+        {
+            #[cfg(not(target_arch = "wasm32"))]
+            re_viewer_context::Clipboard::with(|clipboard| {
+                clipboard.set_text(CODE.into());
+            });
```

Comment:
`re_viewer_context::Clipboard` is for copying images. We should probably remove the `set_text` footgun in it. egui/eframe supports copy-and paste on web:

```suggestion
            ui.ctx().output_mut(|o| o.copied_text = CODE.clone());
```